ExpenseEdit/DeleteExpense.py

```
@@ -0,0 +1,4 @@
+import requests
+class DeleteExpense:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 0

(E302)